pandas/tests/series/indexing/test_setitem.py

```diff
@@ -126,6 +126,14 @@ def test_setitem_boolean_different_order(self, string_series):
 
         tm.assert_series_equal(copy, expected)
 
+    def test_setitem_boolean_using_python_list(self):
+        # GH#19406
+        series = Series([None, "b", None])
```

Comment:
call it `s`

Reply:
Please something like ser, no Single letter names